netjsonconfig/backends/base.py

```diff
@@ -107,20 +121,35 @@ def render(self, files=True):
         """
         self.validate()
         output = ''
-        # render config
+
+        # iterate over the available renderers
+        # and using the available configuration
+        # dictionary output the rendered configuration
         for renderer_class in self.renderers:
+
             renderer = renderer_class(self)
+
             additional_output = renderer.render()
+
             # add an additional new line
-            # to separate blocks
+            # to separate blocks of configuration
+            # generated by different renderers
             if output and additional_output:
                 output += '\n'
+
+            # concatenate the render configuration
             output += additional_output
+
+        # are we required to include
```

Comment:
As this is the base renderer and it's process of rendering configuration is not described elsewhere I would like to mantain this comment.